docs/maintain/maintain-guides-async-backing.md

```diff
@@ -214,6 +214,22 @@ This phase consists of changes to your parachain’s runtime that activate async
 
 ![minimum-period](../assets/async/async-backing-minimum-period.png)
 
+6. **Update parameters:** Once the runtime is deployed the final step is to update three parameters!
```

Comment:
In my case these had to be set on local relay to then propagate to parachain, so might be useful to explicitly mention that these are on the relay and only required for local testing?

Reply:
Done, let me know if that's clear.

Reply:
LGTM, nicely done!